website/versioned_docs/version-15.0.0/inmemory_signer.md

```diff
@@ -40,7 +40,7 @@ The `fromSecretKey` method takes a secret that is base58 encoded as a parameter.
 ```js live noInline
 // import { TezosToolkit } from '@taquito/taquito'
 // import { InMemorySigner } from '@taquito/signer'
-// const Tezos = new TezosToolkit('https://limanet.ecadinfra.com');
```

Comment:
Thinking there is no need to update version-15.0.0 doc with nairobi as for version 15 is relevant to lima.

Reply:
Oh. Now I realize. Yes. Will fix that. Thanks